diff --git a/node_modules/typedoc-plugin-docusaurus/dist/theme/partials/metadata.hbs b/node_modules/typedoc-plugin-docusaurus/dist/theme/partials/metadata.hbs
index a1c5741..b38cc52 100644
--- a/node_modules/typedoc-plugin-docusaurus/dist/theme/partials/metadata.hbs
+++ b/node_modules/typedoc-plugin-docusaurus/dist/theme/partials/metadata.hbs
@@ -1,8 +1,8 @@
 {{#if url}}
 ---
-id: {{getIdFromUrl url}}
-title: {{{ getTitleFromName model.name }}}
-sidebar_label: {{{ getSidebarFromName model.name }}}
+id: "{{getIdFromUrl url}}"
+title: "{{{ getTitleFromName model.name }}}"
+sidebar_label: "{{{ getSidebarFromName model.name }}}"
 ---
 
 {{/if}}
